tests/MyGame/Example/Monster.cs

```diff
@@ -129,6 +129,33 @@ public sealed class Monster : Table {
     return new Offset<Monster>(o);
   }
   public static void FinishMonsterBuffer(FlatBufferBuilder builder, Offset<Monster> offset) { builder.Finish(offset.Value, "MONS"); }
+  
+  public static int KeysCompare(string o1, string o2) { return o1.CompareTo(o2); }
```

Comment:
In Java `compareTo` doesn't work with primitives. I can use `compare` method of the corresponding wrapper class or make the comparison in the lambda expression, if it would be better.